lib/trino-hdfs/src/test/java/io/trino/hdfs/TestFileSystemCache.java

```diff
@@ -58,7 +57,7 @@ public void testFileSystemCache()
             throws IOException
     {
         HdfsEnvironment environment = new HdfsEnvironment(
-                new DynamicHdfsConfiguration(new HdfsConfigurationInitializer(new HdfsConfig()), ImmutableSet.of()),
+                new DynamicHdfsConfiguration(new HdfsConfigurationInitializer(new HdfsConfig()), ImmutableSet.of(new HdfsConfigurationProvider(new HdfsConfig()))),
```

Comment:
is `new HdfsConfigurationProvider(new HdfsConfig())` no-op?
what does it do?

Reply:
It is a no-op. I included it in the tests here to show that it is a no-op.